gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1-bin.zip
```

Comment:
By relying on the upstream site instead of our artifactory mirror, this adds another external dependency to our build. I worked around this when installing Gradle in the Docker image by first trying to download from artifactory, and then falling back to upstream if that failed, which admittedly was pretty hacky and wouldn't work here anyway.

I'm not opposed to this if we are still running some parts of the build outside the Delphix network, but I just thought that I would point it out.

Reply:
Can we configure this based on if we're running in the container or not? I was planning to still leverage the container for running the style checks (so we get faster feedback via the PR status), so we'd still need gradle in TravisCI. So, if we're in the container, use upstream.. if not, use artifactory.. is this possible?

Alternatively, we can remove the use of the container completely, and run the style checks via the Jenkins job (i.e. only support the artifactory URL).

@jgallag88 do you have a preference?

Reply:
> if we're in the container, use upstream.. if not, use artifactory.. is this possible?

That sounds reasonable to me. Since the container already uses upstream, all we would have to do is update this url to point to artifactory, right?

Reply:
I guess you're right. The container can pull gradle just like it does now, and we can still use `gradle` (instead of `gradlew`) in Travis. I'll go with that.